src/main/webapp/index.html

```diff
@@ -13,7 +13,7 @@ <h1>Our webapp</h1>
       <p>This is our webapp.</p>
       <p>Click here to write a journal entry:</p>
 
-      <form action= "/my-data-url" id="my-form" method="POST" enctype="multipart/form-data">
+      <form action= "/my-data-url" method="POST" >
```

Comment:
Removing the space after the "action=" and before the last ">" would help format the form layout better. Below on line 20, the extra space before the "/>" would be helpful too.

Reply:
Done :)